src/runtime/task/task.h

```diff
@@ -359,15 +359,18 @@ class timer_task : public task
     void exec() override;
     void enqueue() override;
 
+    void update_interval(int interval_ms);
+
 protected:
     void clear_non_trivial_on_task_end() override { _cb = nullptr; }
 
 private:
-    // ATTENTION: if _interval_milliseconds <= 0, then timer task will just be executed once;
-    // otherwise, timer task will be executed periodically(period = _interval_milliseconds)
-    int _interval_milliseconds;
+    // ATTENTION: if _interval_ms == 0, then timer task will just be executed once;
+    // otherwise, timer task will be executed periodically(period = _interval_ms)
+    int _interval_ms;
```

Comment:
Declared as `uint32_t` ?

Reply:
Good suggestion, but I don't want to change it in this patch. There are many callers have to update if changing it, these changes are not closer to this patch topic.